src/test/didPropsChange.test.ts

```diff
@@ -1,4 +1,3 @@
-import React from 'react'
```

Comment:
This file doesn't actually use any react code, so my editor was pointing out that this was never being used.